apps/client/src/pages/room/Room.page.tsx

```diff
@@ -10,6 +10,15 @@ export function Room(): ReactElement {
       <Text type={TextType.Heading} size={TextSize.Large}>
         Room
       </Text>
+      {/*TODO: Implement PlayerList*/}
+      {/*<div className={cs('my-12')}>*/}
+      {/*  <ListItem*/}
+      {/*    title='JohnDoe'*/}
+      {/*    imgURL='https://avatars.dicebear.com/api/open-peeps/737373.svg'*/}
+      {/*    number={1}*/}
+      {/*    rightText='64 wpm'*/}
+      {/*  />*/}
+      {/*</div>*/}
```

Comment:
Was this commented part intended to be commented ?

Reply:
This page is entirely recreated on PR https://github.com/the-ai-team/razor/pull/95